src/solver/hydro/management/management.cpp

```diff
@@ -161,21 +161,26 @@ void HydroManagement::prepareNetDemand(uint numSpace)
             auto realmonth = study.calendar.months[month].realmonth;
             auto dayYear = study.calendar.hours[hour].dayYear;
 
-            double netdemand
-              = +scratchpad.ts.load[ptchro.Consommation][hour] - scratchpad.miscGenSum[hour]
-                - ror[hour]
-                - ((ModeT != Data::stdmAdequacy) ? scratchpad.mustrunSum[hour]
-                                                 : scratchpad.originalMustrunSum[hour]);
+            double netdemand = 0;
 
             // Aggregated renewable production: wind & solar
             if (parameters.renewableGeneration.isAggregated())
             {
-                netdemand -= scratchpad.ts.solar[ptchro.Solar][hour]
```

Comment:
There is code duplication here (load, miscGen, ror, mustRun), but if this makes non-regression disappear, why not ?